packages/fetchai/connections/ledger/contract_dispatcher.py

```diff
@@ -356,7 +356,7 @@ def _validate_and_call_callable(
         :return: the data generated by the method.
         """
         try:
-            method_to_call = getattr(contract, message.callable)
+            method_to_call = getattr(contract, "default_method_call")
```

Comment:
no this would not change, instead below, in the exception block we can add this and then see if the method is defined on the abi and available via  `contract.default_method_call(...)`